dev/domain-controller/azuredeploy.json

```diff
@@ -35,12 +35,12 @@
                 "description": "The password of the Azure account/service principal with contributor access to the subscription. This account cannot require MFA."
             }
         },
-        "activationCode": {
-            "type": "string",
-            "defaultValue": "xxxx-xxxx-xxxx-xxxx",
-            "metadata": {
-                "description": "The license activation code for the PCoIP CAS licenses"
-            }
+        "registrationCode": {
+          "type": "securestring",
+          "minLength": 21,
+          "metadata": {
+            "description": "PCoIP Standard Agent License Registration Code."
```

Comment:
That's interesting - it's not really the 'standard agent' registration code since the same code can be used for all agent types. Wondering what we should call that later...